src/sparse_merkle_tree/smt.rs

```diff
@@ -2481,6 +2488,51 @@ mod tests {
         );
     }
 
+    #[test]
+    fn test_bitmap_len_with_verify() {
+        let mut data = UpdateData::new_from(Cache::new());
+        let keys = vec!["bbbbc758f6d27e6cf45272937977a748fd88391db679ceda7dc7bf1f005ee879"];
+        let values = vec!["9c12cfdc04c74584d787ac3d23772132c18524bc7ab28dec4219b8fc5b425f70"];
+
+        for i in 0..keys.len() {
+            data.insert(SharedKVPair(
+                &hex::decode(keys[i]).unwrap(),
+                &hex::decode(values[i]).unwrap(),
+            ));
+        }
+
+        let mut tree = SparseMerkleTree::new(&[], KeyLength(32), Default::default());
+        let mut db = smt_db::InMemorySmtDB::default();
+        let root = tree.commit(&mut db, &data).unwrap();
+
+        let mut proof = tree
+            .prove(
+                &mut db,
+                &keys
+                    .iter()
+                    .map(|k| hex::decode(k).unwrap())
+                    .collect::<NestedVec>(),
+            )
+            .unwrap();
+
+        proof.queries[0].bitmap = Arc::new(vec![
+            1u8, 2, 3, 4, 1, 3, 3, 71, 3, 3, 71, 3, 3, 71, 3, 3, 71, 3, 3, 71, 3, 3, 71, 3, 3, 71,
+            3, 3, 71, 3, 3, 71, 3, 3, 7,
+        ]);
```

Comment:
i think we should check the cases
* 32 byte key with 32 byte bitmap
* 32 byte key with 33 byte bitmap

Reply:
Done.